_positions/product-lead.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Product Lead
+permalink: pay-grades/product-lead/
+team: 18F
+active: true
+---
+**GS Grade Level:** 13
```

Comment:
@hestherchang You can use this page for the product lead position listing. I updated the metadata (everything above the `---`), so you can start here! :)